auditlog/models/rule.py

```diff
@@ -265,6 +278,7 @@ def _make_create(self):
         """Instanciate a create method that log its calls."""
         self.ensure_one()
         log_type = self.log_type
+        users_to_exclude_ids = self.mapped("users_to_exclude_ids")
```

Comment:
```suggestion
        users_to_exclude = self.mapped("users_to_exclude_ids")
```
We use the suffix `_ids` for 2many field declaration, but for internal variables better to not use it or we could think it's a list of IDs.